unlock-protocol-com/blog/websub.md

```diff
@@ -0,0 +1,22 @@
+---
+title: Websub
+subTitle: Enabling subscriptions to new locks and new keys!
+authorName: Julien Genestoux
+publishDate: Feb 28, 2022
+description: WebSub is a W3C protocol that provides a common mechanism for communication between publishers of any kind of Web content and their subscribers, based on HTTP web hooks.
+image: /images/blog/websub.png
+---
+
+As developers integrate the Unlock Protocol in their applications, we are getting many feature requests to add APIs and tools that would make their lives easier.
+
+![Websub](/images/blog/websub.png)
+
+Today, we're excited to announce our [WebSub endpoints](https://docs.unlock-protocol.com/unlock/developers/locksmith/webhooks)!
+
+WebSub is a [W3C protocol](https://www.w3.org/TR/websub/) that provides a common mechanism for communication between publishers of any kind of Web content and their subscribers, based on HTTP **web hooks**.
```

Comment:
WebSub [W3C protocol](https://www.w3.org/TR/websub/) provides a common mechanism for communication between publishers of any kind of Web content and their subscribers, based on HTTP **web hooks**.